embark.el

```diff
@@ -3192,9 +3192,13 @@ buffer."
                (name (embark--descriptive-buffer-name 'export))
                (rerun (embark--rerun-function #'embark-export))
                (buffer (save-excursion
-                         (with-current-buffer (funcall exporter candidates)
-                           (rename-buffer name t)
-                           (current-buffer)))))
+                         (let* ((maybe-buf (funcall exporter candidates))
+                                (selected-buf (if (bufferp maybe-buf)
+                                                  maybe-buf
+                                                (current-buffer))))
```

Comment:
>  Iirc we had a similar discussion about the exporter functionality before

Yes we did. I thought this PR might be harmless enough to exist, because there are some thirdparty exporters that don't pop up a buffer, then it will pick the minibuffer instead.

Reply:
It is not harmless since it establishes that the incorrectly implemented (third party?) exporters won't have to be improved. Where are these exporters? They should be patched instead.